tasks/main.yml

```diff
@@ -9,30 +9,41 @@
   register: result
   until: result is succeeded
 
-- name: Cloning Ansible Hub UI
-  git:
-    repo: 'https://github.com/ansible/ansible-hub-ui.git'
-    dest: '{{ pulp_user_home }}/ansible-hub-ui'
-    force: yes
-    version: master
-    update: yes
-
-- name: Install packages based on package.json
-  npm:
-    path: '{{ pulp_user_home }}/ansible-hub-ui'
-
-- name: Building Ansible Hub UI
-  command: npm run build-standalone
-  args:
-    chdir: '{{ pulp_user_home }}/ansible-hub-ui'
+- name: Check if Ansible Hub UI is setup
+  stat:
+    path: '{{ pulp_webserver_static_dir }}galaxy_ui'
+  register: pulp_static_ui_path
+
+- block:
+
+  - name: Cloning Ansible Hub UI
+    git:
```

Comment:
As we discussed, add the tag:
`molecule-idempotence-notest`
Since the repo can be changed during molecule run.